src/overview/index.md

```diff
@@ -1,21 +1,58 @@
 # Ecosystem Overview
 
-There are two approaches for using Rust on Espressif chips:
+There are the following approaches to use Rust on Espressif chips:
 
-1. _With_ the full standard library available (`std`)
-2. _Without_ the standard library available (`no_std`)
+- Using the `std` library, a.k.a Standard library.
+- Using the `core` library (`no_std`), a.k.a. bare metal development.
 
-Both approaches have their advantages and disadvantages, so you should make a decision based on your project's needs. This chapter contains an overview of the two approaches followed by a brief comparison between them.
+Both approaches have their advantages and disadvantages, so you should make a decision based on your project's needs. This chapter contains an overview of the two approaches:
 
-- [Using the Rust Standard Library (`std`)](./using-the-standard-library.html)
-- [Bare Metal (`no_std`)](./bare-metal.html)
-- [Comparing `std` and `no_std`](./comparing-std-and-no_std.html)
+- [Using the Standard Library (`std`)][rust-esp-book-std]
+- [Developing on Bare Metal (`no_std`)][rust-esp-book-no-std]
+
+[rust-esp-book-std]: ./using-the-standard-library.md
+[rust-esp-book-no-std]: ./bare-metal.md
+
+See also the comparison of the different runtimes in [The Embedded Rust Book][embedded-rust-book-intro-std].
+
+[embedded-rust-book-intro-std]: https://docs.rust-embedded.org/book/intro/no-std.html#a-no_std-rust-environment
 
 The [esp-rs organization] on GitHub is home to a number of repositories related to running Rust on Espressif chips. Most of the required crates have their source code hosted here.
 
 > A note on the repository naming convention
 > In the [esp-rs organization] we use the folling wording:
```

Comment:
```suggestion
> A note on the repository naming convention
> 
> In the [esp-rs organization] we use the folling wording:
```

Reply:
`folling wording` -> `follwing wording`

Reply:
Good catch! Thanks! That reminded me of running a quick LanguageTool check on all the chapter!